pkgs/development/python-modules/pslab/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, buildPythonPackage, fetchPypi, isPy27, numpy, pyserial, scipy }:
+
+buildPythonPackage rec {
+  pname = "pslab";
+  version = "2.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0iifjz7j6g982sc1id91d2krrv6zr42v2fzln9kw7nwjzg1j2j3a";
+  };
+
+  disabled = isPy27;
```

Comment:
```suggestion
```
No longer required.

Reply:
how this is no longer required?

Reply:
python2 is now extra https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/python2-packages.nix